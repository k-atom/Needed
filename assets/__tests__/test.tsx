import * as React from "react";
import * as renderer from "react-test-renderer";
import { MemoryRouter } from 'react-router'
import Header from "../ts/components/header";

test('Header', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Header />
    </MemoryRouter>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
