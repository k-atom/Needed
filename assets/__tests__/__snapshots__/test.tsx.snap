// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header 1`] = `
<nav
  className="navbar navbar-light bg-light head__navbar"
>
  <div
    className="navbar-brand menu"
  >
    <i
      className="material-icons hamburger_menu"
    >
      menu
    </i>
  </div>
  <a
    className="navbar-brand"
    href="/"
    onClick={[Function]}
    title="Needed"
  >
    <img
      alt="Needed Logo"
      className="head__logo"
      src="/images/K_Logo_160x160.png"
    />
    Needed
  </a>
</nav>
`;
